vtex-apps/drawer
# Add close drawer button block

#### What is the purpose of this pull request?
Adds a new block to handle the position of the close button inside the drawer.

#### What problem is this solving?
Enables more customizations of the placement and layout of the close button, like in the following image

![image](https://user-images.githubusercontent.com/10223856/76455023-2557de00-63b4-11ea-8120-bd21f0a076f9.png)

#### How should this be manually tested?
Linked in [this workspace](https://lucas--checkoutio.myvtex.com/). You can see that the minicart drawer didn't change with this new structure. You can also see the above customization in [this route](https://lucas--checkoutio.myvtex.com/checkout/#/), by clicking the "View Cart" button.

#### Types of changes
